sdk/tests/deqp/modules/shared/glsTextureTestUtil.js

```diff
@@ -1922,8 +1925,18 @@ glsTextureTestUtil.computeTextureLookupDiffCube = function(
             qpWatchDog_touch(watchDog);*/
 
         for (var px = 0; px < result.getWidth(); px++) {
+            /** @type {Array<number>} */
             var resPix = result.getPixel(px, py);
+            resPix[0] = (resPix[0] - sampleParams.colorBias[0]) / sampleParams.colorScale[0];
```

Comment:
This similar processing repeated six times in this CL.  Could you define an inline helper function so the code looks cleaner without harming perf?